src/utils/workpackageHelper.js

```diff
@@ -89,4 +90,96 @@ export const workpackageHelper = {
 			fileId: workPackage.fileId,
 		}
 	},
+	chunkMultipleSelectedFilesInformation(fileInformation) {
+		// this function chunks 20 files information in an array and returns array of it
+		const chunkedResult = []
+		for (let i = 0; i < fileInformation.length; i += 20) {
+			chunkedResult.push(fileInformation.slice(i, i + 20))
+		}
+		return chunkedResult
+	},
+	getTotalNoOfFilesInAlreadyChunkedFilesInformation(chunkedFilesInformations) {
+		const totalFiles = []
+		for (let i = 0; i < chunkedFilesInformations.length; i++) {
+			totalFiles.push(...chunkedFilesInformations[i])
+		}
+		return totalFiles.length
+	},
+
+	/*
+		Here the chunkedFilesInformation is the array that contains the array of chunked files information in 20
+	*/
+	async linkMultipleFilesToWorkPackageWithChunking(chunkedFilesInformations, selectedWorkpackage, isRemaining, component) {
+		const chunkingInformation = {
+			totalNoOfFilesSelected: this.getTotalNoOfFilesInAlreadyChunkedFilesInformation(chunkedFilesInformations),
+			totalFilesAlreadyLinked: 0,
+			totalFilesNotLinked: this.getTotalNoOfFilesInAlreadyChunkedFilesInformation(chunkedFilesInformations),
+			isChunkingError: false,
```

Comment:
is this an error about chunking or when the transfer did not happen?
```suggestion
			error: false,
```

Reply:
yes it is basically error key when there is some error while linking with chunk. But may be `error` is enough.